docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -73,13 +73,17 @@ include::{sourcedir310}/ehcache-impl/src/test/java/org/ehcache/docs/Tiering.java
 <2> A shortcut to specify 10 entries.
 <3> Only 10 MB allowed. Eviction will occur when full.
```

Comment:
Lets just delete this callout and the referenced code in the sample. Then you can delete the `NOTE:` below as well.

Reply:
Agreed, DONE